docs/user-guide/sdks-using.md

```diff
@@ -0,0 +1,123 @@
+
+   The SDK is downloaded to your computer.
+
+2. Unzip the SDK folder, which contains the packages for each set of functionality (such as z/OS Jobs). Copy each file that you want to install and paste them into your project directory.
+
+3. In a command-line window, navigate to your project directory. Issue one of the following commands:
+
+   - To install a Node package: `npm install <fileName>.tgz`
+   - To install a Python package: `pip install <fileName>.whl`
+
+   The packages are installed and you can call the APIs from within your project.
+
+### Install SDK from online registry
+
+Define the packages as dependencies in your project and pull them from an online registry.
+
+**Follow these steps:**
+
+1. In command-line window, navigate to your project directory. Issue the following command to install a package from the registry:
+
+   - To import a Node package: `npm install <sdkPackageName>`
+   - To import a Python package: `pip install <sdkPackageName>`
+
+    The package is installed and defined in your `package.json` or `requirements.txt` file.
+
+2. **(Optional)** You might want to automatically update the SDK version when updates become available, or you might want to prevent automatic updates. Use [npm semver](https://docs.npmjs.com/misc/semver#x-ranges-12x-1x-12-) to define the versioning scheme.
+
+    The packages are installed and you can call the APIs from within your project.
+
+## Using
+
+The following examples demonstrate how to call the API endpoints:
+
+**Node Example:**
+
+After you install the package in your project, create a session object to handle communication with z/OSMF. This example shows how to import the z/OS Files APIs and define your connection details in the session object:
+
+```javascript
+import { Download } from "@zowe/zos-files-for-zowe-sdk";
+
+const session = new Session({
+    hostname: '<host address>',
+    user: '<zosmf user>',
+    password: '<zosmf password>'
+});
+```
+
+To use a Zowe CLI z/OSMF profile instead of manual authentication, provide the profile name when you create the object:
+
+```javascript
+import { CliProfileManager } from "@zowe/imperative";
+import { Submit } from "@zowe/zos-jobs-for-zowe-sdk";
+
+const profileManager = new CliProfileManager({
+  	profileRootDirectory: require("path").join(require("os").homedir(), ".zowe", "profiles"),
+  	type: "zosmf"
+});
+const profile = (await profileManager.load({ loadDefault: true })).profile;
+```
+
+**Python Example**
+
+After you install the package in your project,  import the `ZoweSDK` class. Create an object to handle communication with z/OSMF:
+
+```python
+from zowe_sdk import ZoweSDK
+
+z = ZoweSDK(zosmf_host='<host address>', zosmf_user='<zosmf user>', zosmf_password='<zosmf password>')
+```
+
+To use a Zowe CLI z/OSMF profile instead of manual authentication, provide a profile name when you create the object:
+
+```python
+from zowe_sdk import ZoweSDK
+z = ZoweSDK(zosmf_profile='<profile name>')
+```
```

Comment:
```suggestion
After you install the package in your project,  import the class for the required sub-package (i.e `Console` class for z/OS Console commands). Create a dictionary to handle communication with z/OSMF:

```python
from zowe.zos_console_for_zowe_sdk import Console

connection = {
    "host_url": "'<host address>'",
    "user": "<user>",
    "password": "<password>",

}
my_console = Console(connection)
```

To use a Zowe CLI z/OSMF profile instead of manual authentication, provide a profile name when you create the object:

```python
from zowe.zos_console_for_zowe_sdk import Console

connection = {
    "plugin_profile": "<profile name>"
}
my_console = Console(connection)
```
```